dg-common/src/main/scala/org/finra/datagenerator/common/Helpers/RetryHelper.scala

```diff
@@ -40,15 +40,21 @@ object RetryHelper {
               (handlingCode: => Unit = () => ()): T = {
     var result: Option[T] = None
     var left = maxTries
-    while(!result.isDefined) {
+    while (!result.isDefined) {
       left = left - 1
 
       // try/catch{case...} doesn't seem to support dynamic exception types, so using handling block instead.
```

Comment:
It's not commented code -- it's explanation for why I used handling block instead of try/catch.